bertopic/_bertopic.py

```diff
@@ -3237,41 +3237,41 @@ def merge_models(cls, models, min_similarity: float = .7, embedding_model=None):
 
             # Extract new topics
             new_topics = sorted([index - selected_topics["_outliers"] for index, sim in enumerate(sims) if sim < min_similarity])
-            max_topic = max(set(merged_topics["topics"]))
+            max_topic = max(set(merged_topics["topics"])) + 1
```

Comment:
Hi first off I just want to say that I love this library and that I appreciate all your hardwork on it.

I've patched my `merge_models()` function with the changes from this PR. I'm seeing a couple of topics with 'NaN' Names but non zero Count when I call`topic_info()` on the merged model. 
I think it may be due to `max_topic = max(set(merged_topics["topics"])) + 1`, should this just be `max_topic = max(set(merged_topics["topics"]))`?

Reply:
You are absolutely right! I had done that in my original code but it seems I forgot to transfer it over to this PR. A rather silly/lazy mistake. I just changed it and hopefully, it should work now.

Reply:
Sorry forgot to say, works now - thank you!

Reply:
@VinceMu Awesome, thanks for checking!